server/src/main/java/org/opensearch/cluster/coordination/CoordinationState.java

```diff
@@ -449,6 +453,7 @@ public PublishResponse handlePublishRequest(PublishRequest publishRequest) {
             clusterState.version(),
             clusterState.term()
         );
+        logger.info("Setting last accepted state : term - {}, version - {}", clusterState.term(), clusterState.version());
```

Comment:
This will pollute the logs I think, we can keep it in debug mode.

Reply:
Lets use on demand logger primitives () -> {}

Reply:
@Arpit-Bandejiya I will change it to debug.
@Bukhtawar Could you point me to a code reference for on demand logger primitives?

Reply:
https://logging.apache.org/log4j/2.x/javadoc/log4j-api/org/apache/logging/log4j/util/class-use/Supplier.html

Reply:
Lets move to debug logging